src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -2025,7 +2037,7 @@ public boolean scheduleBuild(int quietPeriod, Cause c, Action... actions) {
      * {@inheritDoc}
      */
     public void checkAbortPermission() {
-        checkPermission(AbstractProject.ABORT);
+        checkPermission(Item.CANCEL);
```

Comment:
Well, this is part of the `Queue.Task` interface, so it cannot really be changed.